### PR TITLE
Fix docs for GraphQL.NET 3.0.0 release

### PR DESCRIPTION
**Pull request by sungam3r**
See https://github.com/graphql-dotnet/graphql-dotnet/issues/1843

### CODE REVIEW

**Comment by sungam3r** [comment]
@IvanGoncharov @leebyron Could you please review? GraphQL.NET received a long-awaited update to version 3.0.0 and it took a little change in the documentation.

**Comment by JVimes** [comment]
@sungam3r, I believe there are a few issues:
* I don't think these changes compile. Try following your documentation as if you were a new user.
* Your changes don't fix the exception in the [original issue](https://github.com/graphql-dotnet/graphql-dotnet/issues/1843). See [this comment](https://github.com/graphql-dotnet/graphql-dotnet/issues/1843#issuecomment-691457828).
* Shouldn't this merge request change the [source documentation](https://github.com/graphql-dotnet/graphql-dotnet/tree/master/docs2)? Did you follow the [publishing instructions](https://github.com/graphql-dotnet/graphql-dotnet/blob/master/docs2/README.md#publishing)?

**Comment by sungam3r** [comment]
> I don't think these changes compile. Try following your documentation as if you were a new user.
изображение

> Your changes don't fix the exception in the original issue. See this comment.

😕 

> Shouldn't this merge request change the source documentation? Did you follow the publishing instructions?

Our own documentation http://graphql-dotnet.github.io is now updated automatically after migrating to GitHub Actions.

File: site/code/index.html.js [hunk 1]
```diff
@@ -53,6 +53,13 @@ In addition to the GraphQL [reference implementations in JavaScript](#javascript
 using System;
 using GraphQL;
 using GraphQL.Types;
+using GraphQL.SystemTextJson;
+
+// You can use GraphQL.NewtonsoftJson with .NET Core 3+, just be aware it lacks async writing 
+// capabilities so writing to an ASP.NET Core 3.0 HttpResponse.Body will require you to set 
+// AllowSynchronousIO to true which isn't recommended.
```
Comment (joemcbride) [inline site/code/index.html.js:60]:
```
AllowSynchronousIO ...
```

This comment pertains to ASP.NET Core, which isn't this example of a Console application.  I don't think we need to add this comment here.

**Review by joemcbride — CHANGES_REQUESTED** [review]

**Comment by joemcbride** [comment]
@sungam3r I think it's much more intuitive to use async/await.

**Comment by sungam3r** [comment]
OK, changed.

**Comment by JVimes** [comment]
@sungam3r What version are you on? It does not work for me:

image

**Comment by Shane32** [comment]
You need to import a serialization library as well, such as `GraphQL.SystemTextJson`, or else pass an implementation of `IDocumentWriter` to the extension method.

**Comment by sungam3r** [comment]
@JVimes 
изображение

OK?

**Review by joemcbride — APPROVED** [review]

**Comment by JVimes** [comment]
@Shane32 and @sunapi386 -- No, GraphQL.SystemTextJson is *not* part of the hello world code sample! Look for yourself [here](https://github.com/graphql-dotnet/graphql-dotnet/blob/master/docs2/site/docs/getting-started/introduction.md) and [here](https://graphql.org/code/#c-net) and [here](https://graphql-dotnet.github.io/docs/getting-started/introduction/). The code sample does not have a `using` statement to bring in the ExecuteAsync extension method. Did you understand the [bug report](https://github.com/graphql-dotnet/graphql-dotnet/issues/1843)? The code sample does not work!

**Comment by Shane32** [comment]
> No, GraphQL.SystemTextJson is _not_ part of the hello world code sample! Look for yourself [here](https://github.com/graphql-dotnet/graphql-dotnet/blob/master/docs2/site/docs/getting-started/introduction.md) and [here](https://graphql.org/code/#c-net) and [here](https://graphql-dotnet.github.io/docs/getting-started/introduction/). The code sample does not have a `using` statement to bring in the ExecuteAsync extension method. Did you understand the [bug report](https://github.com/graphql-dotnet/graphql-dotnet/issues/1843)? The code sample does not work.

You're correct.  The documentation is wrong, both in this library, and in the graphql-dotnet project samples, for version 3.0.  Keep in mind that unlike version 2.4, the json serialization library has been split off from the main project into separate nuget packages, and you will need one of them to make any practical use of the library - you'd need either `GraphQL.SystemTextJson` or `GraphQL.NewtonsoftJson` for a sample as small as the "hello world" sample shown.

In addition to the PR here to fix the sample shown [here](https://graphql.org/code/#c-net), we will need a PR in the graphql-dotnet repo to fix the file [here](https://github.com/graphql-dotnet/graphql-dotnet/blob/master/docs2/site/docs/getting-started/introduction.md) which when published will update the documentation [here](https://graphql-dotnet.github.io/docs/getting-started/introduction/).  Thank you for pointing this out to us.

**Comment by JVimes** [comment]
Thank you. I think this will really help.

**Comment by orta** [comment]
I'm going to close this PR as these changes were manually merged upstream by @ardatan in #936 - thanks for updating the list. 

Note: We now [have markdown files](https://github.com/graphql/graphql.github.io/tree/source/src/content/code) which allow for longer descriptions on the new code page: https://graphql.org/code/ - so it'd be awesome if you added some code samples etc to help any project on the page shine.

**Pull request closed without merging**